Cargo.toml

```diff
@@ -32,3 +32,9 @@ panic = "abort"
 [profile.phd]
 inherits = "dev"
 panic = "unwind"
+
+[patch."http://github.com/oxidecomputer/rfb"]
+# Double / here is intentional to defeat cargo's git canonicalization and allow us to
+# patch a crate using the same source.
+# Temp until https://github.com/oxidecomputer/rfb/pull/9 lands
+rfb = { git = "http://github.com/oxidecomputer//rfb", branch = "luqmana/vnc-stop" }
```

Comment:
Perhaps a good time to make the cargo dep specify an explicit rev like crucible?

Reply:
Yup, will do once the RFB PR lands.